docs/source/03_tutorial/06_visualise_pipeline.md

```diff
@@ -96,3 +96,104 @@ kedro viz --load-file my_shareable_pipeline.json
 ```
 
 And this will visualise the pipeline visualisation saved as `my_shareable_pipeline.json`.
+
+
+## Visualise Plotly charts in Kedro-Viz
+
+[Plotly](https://plotly.com/python/) is a free and open source Python library that allows you to make interactive, publication quality graphs. With Plotly integration on Kedro-Viz, you can output your interactive charts as part of your pipeline visualisation.
+
+Kedro-Viz aims to help users communicate different aspects of their workflow through an interactive flowchart. With Plotly integration, we take one step further in this direction to allow our users to effectively share their data insights while exploring the pipeline.
+
+We have also used the Plotly integration to allow users to [visualise metrics from experiments](https://kedro.readthedocs.io/en/stable/08_logging/02_experiment_tracking.html?highlight=experiment%20tracking).
+
+Note: Kedro's Plotly integration only supports [Plotly Express](https://plotly.com/python/plotly-express/) charts.
+
+You can view Plotly charts in Kedro-Viz when you use Kedro's plotly datasets.
+
+There are two types of plotly datasets in Kedro :
+- [plotly.PlotlyDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.PlotlyDataSet.html#kedro.extras.datasets.plotly.PlotlyDataSet) - To use this dataset you need to configure your plot in the `catalog.yml`.
+
+Below is an example of how to visualise plots on Kedro-Viz using `plotly.PlotlyDataSet`
+
+```python
```

Comment:
I think it would be worth adding what files this snippet and the `yml` one below should be added to.